.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build-vmake:
```

Comment:
I'm guessing this is intended to be `build-cmake`?

Reply:
Yes, force pushed with a fix.